packages/ui/src/components/va-input/components/VaInputWrapper/_variables.scss

```diff
@@ -0,0 +1,20 @@
+:root {
```

Comment:
We will definitely miss this CSS variables in docs. Consider adding VaInputWrapper CSS Variables block in VaInput.